src/net/wooga/jenkins/pipeline/config/Platform.groovy

```diff
@@ -26,8 +26,10 @@ class Platform {
     }
 
     static Platform forWDK(BuildVersion buildVersion, String buildOS, Map config, boolean isMain) {
-        def unityEnv = ["UVM_UNITY_VERSION=${buildVersion.version}",
-                        "UNITY_LOG_CATEGORY=check-${buildVersion.version}"]
+        def unityEnv = ["UNITY_LOG_CATEGORY=check-${buildVersion.version}"]
+        if(!buildVersion.isAutoVersion()) {
+            unityEnv.add("UVM_UNITY_VERSION=${buildVersion.version}")
+        }

```

Comment:
Since we are in control here it could make it more readable if `buildVersion` has a getter like: `hasVersion`.

Reply:
and Done :)